.github/workflows/maven-verify.yml

```diff
@@ -27,4 +27,5 @@ jobs:
     uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v2
     with:
       jdk-distribution-matrix: '[ "temurin", "zulu", "microsoft", "liberica", "adopt-openj9" ]'
```

Comment:
I know it's not part of this change, but do we really expect big differences between let's say Liberia, Zulu, Microsoft and Temurin? I do see why OpenJ9 is on the list, but all the others are regular Hotspot JVM's, aren't they?

Reply:
probably not :) but as I already got differences (for javadoc) between distributions

Reply:
@mthmulders what would be your suggestion to limit to which?

Reply:
Usually, I'd say "a Hotspot VM and an OpenJ9 VM". For OpenJ9, we don't have my choices; for Hotspot, I guess Temurin or maybe Zulu?

But if you're saying there are differences (e.g. in generated Javadoc) and they matter for the m-c-p, then we should include them.